### PR TITLE
Preload bower and gulp in build image

### PR DESCRIPTION
**Pull request by troydai**

### CODE REVIEW

**Comment by glennc** [comment]
Seems fine. Can we stop circleci from running?

**Comment by troydai** [comment]
I'll address the circleci after this PR.

File: 1.0.1/jessie/build/Dockerfile [hunk 1]
```diff
@@ -21,4 +21,9 @@ RUN curl -o /tmp/packagescache.tar.gz http://dist.asp.net/packagecache/aspnetcor
     # restore 1.0.1 ASP.NET packages
     cd /tmp && dotnet restore packagescache.project.json && \
     rm /tmp/packagescache.project.json && \
+    # set up node
+    apt-get -qq update && \
+    apt-get -qqy install node npm && \
```
Comment (shirhatti) [inline 1.0.1/jessie/build/Dockerfile]: Isn't that the wrong [node](https://packages.debian.org/jessie/node) ?

Here's what the official node docker install does- [Dockerfile](https://github.com/nodejs/docker-node/blob/5ad063e3ba340743e394114d3038d0fd4e0fe570/4.5/wheezy/Dockerfile#L21-L27)

Comment (troydai) [inline 1.0.1/jessie/build/Dockerfile]: Does this node comes with npm?

Comment (shirhatti) [inline 1.0.1/jessie/build/Dockerfile]: Yes. It comes with ["nodejs"](https://packages.debian.org/jessie/nodejs) but that version is waaay out of date

Comment (troydai) [inline 1.0.1/jessie/build/Dockerfile]: ok, let me try the _right way_.

Comment (shirhatti) [inline 1.0.1/jessie/build/Dockerfile]: Another option is to add the Nodesource PPA and then install nodejs
```
curl -sL https://deb.nodesource.com/setup_4.x | sudo -E bash -
sudo apt-get install -y nodejs
```

Comment (troydai) [inline 1.0.1/jessie/build/Dockerfile]: https://github.com/aspnet/aspnet-docker/pull/151

**Pull request merged**